### PR TITLE
[GD-2103] Replace webpack with vite

### PR DESCRIPTION
**Pull request by cjasonbush**

### CODE REVIEW

File: packages/frontend/src/admin-pages/components/nonprofit/yourPage/tabs/Media.vue [hunk 1]
```diff
@@ -259,6 +259,7 @@ export default {
         vue.clearModals()
       }).catch(function (err) {
         vue.clearModals()
+        console.log('ERR', err)
```
Comment (dylanmmatthews) [inline packages/frontend/src/admin-pages/components/nonprofit/yourPage/tabs/Media.vue:262]: do we want this to stay here?

**Review by dylanmmatthews — CHANGES_REQUESTED** [review]
haven't dove fully into vite stuff yet

File: packages/frontend/src/admin-pages/components/header/Navigation.vue [hunk 2]
```diff
@@ -158,19 +158,19 @@ export default {
       event.preventDefault()
       const vue = this
       if (vue.displayingMenu) {
-        $(vue.$refs.oMenubarPopupParent).removeClass('o-menubar-popup-parent--active')
-        $(vue.$refs.oMenubarPopup).fadeOut(200)
+        window.$(vue.$refs.oMenubarPopupParent).removeClass('o-menubar-popup-parent--active')
```
Comment (dylanmmatthews) [inline packages/frontend/src/admin-pages/components/header/Navigation.vue:161]: I don't think the `window.` is needed at all in this page

File: packages/frontend/src/public-pages/components/modals/Modals.vue [hunk 3]
```diff
@@ -32,6 +32,8 @@ import ComponentDonationTiersModal from './donations/DonationTiersModal.vue'
 import ComponentMenuModal from './MenuModal.vue'
 import ComponentSpinnerModal from './SpinnerModal.vue'
 
+import $ from 'jquery'
```
Comment (dylanmmatthews) [inline packages/frontend/src/public-pages/components/modals/Modals.vue:35]: i don't think this is needed either

**Review by dylanmmatthews — COMMENTED** [review]
cleanup

File: packages/frontend/src/admin-pages/components/admin/pages/custom/CustomPagesEdit.vue [hunk 4]
```diff
@@ -182,7 +182,7 @@
 </template>
 
 <script>
-import { getContentKeys, getSettingKeys } from './../../../../helpers/content'
+import { getSettingKeys, getContentKeys } from '../../../../helpers/content.js'
```
Comment (dylanmmatthews) [inline packages/frontend/src/admin-pages/components/admin/pages/custom/CustomPagesEdit.vue:185]: does this need `.js`?

**Review by dylanmmatthews — COMMENTED** [review]
I have a few unanswered questions from the initial go through as well

File: packages/frontend/src/admin-pages/components/nonprofit/yourPage/tabs/Media.vue (hunk 1 above)
Comment (cjasonbush) [reply packages/frontend/src/admin-pages/components/nonprofit/yourPage/tabs/Media.vue:261]: Nope.

File: packages/frontend/src/admin-pages/components/admin/pages/custom/CustomPagesEdit.vue (hunk 4 above)
Comment (cjasonbush) [reply packages/frontend/src/admin-pages/components/admin/pages/custom/CustomPagesEdit.vue:185]: Jira added that bit for me... good catch

Comment (cjasonbush) [reply packages/frontend/src/admin-pages/components/admin/pages/custom/CustomPagesEdit.vue:185]: I removed all of the `.js's`

File: packages/frontend/src/public-pages/components/modals/Modals.vue (hunk 3 above)
Comment (cjasonbush) [reply packages/frontend/src/public-pages/components/modals/Modals.vue:35]: removed.

File: packages/frontend/src/admin-pages/components/header/Navigation.vue (hunk 2 above)
Comment (cjasonbush) [reply packages/frontend/src/admin-pages/components/header/Navigation.vue:161]: I removed those

File: packages/frontend/src/admin-pages/components/nonprofit/yourPage/tabs/Media.vue (hunk 1 above)
Comment (cjasonbush) [reply packages/frontend/src/admin-pages/components/nonprofit/yourPage/tabs/Media.vue:261]: I remove that one, and one earlier in the file.

**Review by dylanmmatthews — APPROVED** [review]

File: packages/frontend/src/public-pages/assets/js/jquery.fireSlider.min.cjs [hunk 5]
```diff
@@ -0,0 +1 @@
...es(),a.calculatePositions(a.state.currentSlide),a.slides.dequeue(a.options.effect)}},c.prototype.Effects={transitions:{slideInOut:"slideInOut",fadeInOut:"fadeInOut"},register:function(a,b){this.transitions[a]=a,this[a]=b},slideInOut:function(a,b){var c=b.snapping?0:b.speed;("forward"===b.direction||"backward"===b.direction)&&a.velocity({translateX:[b.nextPos+"%",b.currPos+"%"]},{duration:c,queue:b.effect,easing:b.easing}),("up"===b.direction||"down"===b.direction)&&a.velocity({translateY:[b.nextPos+"%",b.currPos+"%"]},{duration:c,queue:b.effect,easing:b.easing})},fadeInOut:function(a,b){var c=a.clone();a.parent().append(c),a.velocity({translateX:[b.nextPos+"%",b.nextPos+"%"]},{duration:b.speed,queue:b.effect,begin:function(){c.velocity({opacity:[0,1],zIndex:[1,1]},{duration:b.speed,easing:b.easing})},complete:function(){c.remove()}})},route:function(a,b){var c=b.effect;"undefined"!=typeof this.transitions[c]&&"function"==typeof this[c]&&this[c](a,b)}},a.fn.fireSlider=function(b){return this.each(function(){a.data(this,"fireSlider")&&(a(this).data("fireSlider").destroy(),a(this).removeData("fireSlider")),a.data(this,"fireSlider",new c(this,b))})},b.fireSlider=c})(jQuery,window)})();
```
Comment (mkeiser82) [inline packages/frontend/src/public-pages/assets/js/jquery.fireSlider.min.cjs:1]: Are we loading velocity from the cdn? I.e. can we remove it here?

File: packages/frontend/src/public-pages/assets/js/jquery.payment.min.cjs [hunk 6]
```diff
@@ -0,0 +1 @@
...):!1):!1},t.payment.validateCardExpiry=function(e,n){var r,a,o;return"object"==typeof e&&"month"in e&&(o=e,e=o.month,n=o.year),e&&n?(e=t.trim(e),n=t.trim(n),/^\d+$/.test(e)&&/^\d+$/.test(n)&&e>=1&&12>=e?(2===n.length&&(n=70>n?"20"+n:"19"+n),4!==n.length?!1:(a=new Date(n,e),r=new Date,a.setMonth(a.getMonth()-1),a.setMonth(a.getMonth()+1,1),a>r)):!1):!1},t.payment.validateCardCVC=function(e,r){var a,o;return e=t.trim(e),/^\d+$/.test(e)?(a=n(r),null!=a?(o=e.length,k.call(a.cvcLength,o)>=0):e.length>=3&&e.length<=4):!1},t.payment.cardType=function(t){var n;return t?(null!=(n=e(t))?n.type:void 0)||null:null},t.payment.formatCardNumber=function(n){var r,a,o,i;return n=n.replace(/\D/g,""),(r=e(n))?(o=r.length[r.length.length-1],n=n.slice(0,o),r.format.global?null!=(i=n.match(r.format))?i.join(" "):void 0:(a=r.format.exec(n),null!=a?(a.shift(),a=t.grep(a,function(t){return t}),a.join(" ")):void 0)):n},t.payment.formatExpiry=function(t){var e,n,r,a;return(n=t.match(/^\D*(\d{1,2})(\D+)?(\d{1,4})?/))?(e=n[1]||"",r=n[2]||"",a=n[3]||"",a.length>0?r=" / ":" /"===r?(e=e.substring(0,1),r=""):2===e.length||r.length>0?r=" / ":1===e.length&&"0"!==e&&"1"!==e&&(e="0"+e,r=" / "),e+r+a):""}}).call(this);
```
Comment (mkeiser82) [inline packages/frontend/src/public-pages/assets/js/jquery.payment.min.cjs:1]: Are we loading velocity from the cdn? I.e. can we remove it here?

**Review by mkeiser82 — CHANGES_REQUESTED** [review]

File: packages/frontend/.gitignore [hunk 7]
```diff
@@ -14,4 +14,7 @@ build/
 
 # Ignore environment variables
 .env
-config/*.json
\ No newline at end of file
+config/*.json
+
+src/admin-pages/settings.json
+src/public-pages/settings.json
```
Comment (mkeiser82) [inline packages/frontend/.gitignore:20]: We also need to add `config/custom.css` as ignored here.  It shows up as an untracked file after e.g. `npm run frontend:build`.

**Review by mkeiser82 — CHANGES_REQUESTED** [review]

**Review by mkeiser82 — APPROVED** [review]

**Pull request closed without merging**